prow/pluginhelp/hook/hook.go

```diff
@@ -303,6 +304,22 @@ func (oa *orgAgent) orgToReposMap(config *plugins.Configuration) map[string]sets
 	return oa.orgToRepos
 }
 
+func reposForOrgOrUser(ghc githubClient, orgOrUser string) ([]github.Repo, error) {
+	var errs [2]error
+	inError := map[bool]string{true: "user", false: "org"}
+
+	// Check as an org first, it is more likely in normal use case
+	for i, isUser := range []bool{false, true} {
+		if repos, err := ghc.GetRepos(orgOrUser, isUser); err == nil {
+			return repos, nil
+		} else {
+			errs[i] = fmt.Errorf("failed to get repos for %s %s: %w", inError[isUser], orgOrUser, err)
```

Comment:
Should this error be swallowed when the use case is user? Or in another word, does `GetRepos(<user>, false)` return error?

Reply:
I'm not sure I understand the question correctly. When we call `GetRepos(<user>, false)` (that's what the current code does) we get a 404 error. The `false` results in getting repos for an **org**, and there isn't an org with that name. In the current state this gets logged as an error (that's how I noticed it) and it is skipped.

So the new wrapper always does one of the following: Either one of the two attempts passes and no error is returned, or both fail and we return an aggregate of the two errors that we hit: one of them will always be a 404 and one will the the "real" one.

Reply:
I see, sorry missed the return above.